CHANGELOG.md

```diff
@@ -4,6 +4,18 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Added
+- added scripts 'check-all-gpg-keys-for-expiry.rb' and its spec script 'check_all_gpg_keys_for_expiry_spec.rb'.
+
+### Changed
+- moved spec_helper.rb from test to spec folder for consistency.
+- updated spec_helper.rb to provide classes and tools for testing check-all-gpg-keys-for-expiry.
+- updated license to include Crown Copyright.
```

Comment:
this should be removed as we are not changing the license.

Reply:
disregard I see that the LICENSE file has not been changed to crown but the new check is which is fine as you state it is released with the same terms as MIT.